ppl/archive/chunk/chunk.go

```diff
@@ -99,20 +100,24 @@ func ChunkFileName(id ksuid.KSUID) string {
 	return fmt.Sprintf("%s-%s.zng", FileKindData, id)
 }
 
-// ZarDir returns a URI for a directory specific to this data file, expected
+// ZarPath returns a URI for a directory specific to this data file, expected
 // to hold microindexes or other files associated with this chunk's data.
-func (c Chunk) ZarDir() iosrc.URI {
+func (c Chunk) ZarPath() iosrc.URI {
```

Comment:
Please don't change this name; add a new method with a new name for new functionality.